app/controllers/sessions_controller.rb

```diff
@@ -6,11 +6,19 @@ def new
   def create
     user = User.find_by(email: params[:email])
     login(user)
-    
+
     session[:user_id] = user.id
     flash[:success] = "Welcome, #{user.name}, you are logged in!"
   end
 
+  def destroy
+    User.find(session[:user_id]).destroy
```

Comment:
It looks like the user will get destroyed here, but we only want to destroy the session, not trying to destroy an entire user.